components/table-list/table/head.js

```diff
@@ -2,8 +2,12 @@ import PropTypes from 'prop-types'
 import {ArrowDown} from 'react-feather'
 import Header from './header'
 
-const order = (a, b) => (
-  <div style={{display: 'inline-flex'}}>
+const order = (a, b, direction) => (
+  <button
+    type='button'
+    style={{display: 'inline-flex', border: 'none', background: 'none', color: 'white'}}
```

Comment:
Évitons le `style in-line` quand cela est possible, c'est une pratique assez dépréciée.